client/a8c-for-agencies/sections/migrations/consolidated-commissions/index.tsx

```diff
@@ -1,34 +1,44 @@
 import { Card } from '@automattic/components';
 import { useTranslate } from 'i18n-calypso';
+import type { MigrationCommissionItem } from '../types';
 
 import './style.scss';
 
-const getCurrentQuarter = () => {
-	const currentMonth = new Date().getMonth();
-	const quarter = Math.ceil( ( currentMonth + 1 ) / 3 );
-	return `Q${ quarter }`;
+const getQuarter = ( date = new Date() ) => {
+	const currentMonth = date.getMonth();
+	return Math.ceil( ( currentMonth + 1 ) / 3 );
 };
 
-export default function MigrationsConsolidatedCommissions() {
+export default function MigrationsConsolidatedCommissions( {
+	items,
+}: {
+	items: MigrationCommissionItem[];
+} ) {
 	const translate = useTranslate();
 
-	const data = {
-		migrationCommissions: 300,
-		sitesPendingReview: 2,
-	}; // FIXME: Replace with real data
+	const migrationCommissions =
+		items.filter(
+			( item ) =>
+				// Consider only confirmed migrations for the current quarter
+				item.reviewStatus === 'confirmed' && getQuarter( item.migratedOn ) === getQuarter()
+		).length * 100; // FIXME: Consider the maximum commission value
+
+	const sitesPendingReview = items.filter( ( item ) => item.reviewStatus === 'pending' ).length;
+
+	const currentQuarter = getQuarter();
 
 	return (
 		<div className="consolidated-commissions">
 			<Card compact>
-				<div className="consolidated-commissions__value"> ${ data.migrationCommissions }</div>
+				<div className="consolidated-commissions__value"> ${ migrationCommissions }</div>
 				<div className="consolidated-commissions__label">
 					{ translate( 'Migration commissions expected in %(currentQuarter)s', {
-						args: { currentQuarter: getCurrentQuarter() },
+						args: { currentQuarter: `Q${ currentQuarter }` },
```

Comment:
Should we include the Q in the translatable string? Other locales may lack the appropriate context for what Q represents here.

Reply:
Thanks for the review, @jkguidaven!

Good point. I have updated it now: https://github.com/Automattic/wp-calypso/pull/96234/commits/d37ea62237d262de1ffec657d985b5a598138de4